channel/src/main/java/com/synopsys/integration/alert/channel/azureboards2/util/AzureBoardsWorkItemExtractionUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * channel
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.channel.azureboards2.util;
+
+import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.alert.common.message.model.LinkableItem;
+import com.synopsys.integration.azure.boards.common.service.workitem.response.WorkItemFieldsWrapper;
+import com.synopsys.integration.azure.boards.common.util.AzureFieldDefinition;
+
+public final class AzureBoardsWorkItemExtractionUtils {
```

Comment:
This class is necessary for reconstructing a bom component when a project/project-version notification is being processed. Project and project-version notifications are only used to resolve issues, so these fields will only be used briefly for consistency.